oauth2-proxy.service.example

```diff
@@ -1,22 +1,33 @@
-# Systemd service file for oauth2-proxy daemon
-#
-# Date: Feb 9, 2016
-# Author: Srdjan Grubor <sgnn7@sgnn7.org>
-
 [Unit]
 Description=oauth2-proxy daemon service
-After=syslog.target network.target
+After=network.target network-online.target nss-lookup.target basic.target
+Wants=network-online.target nss-lookup.target
+StartLimitIntervalSec=30
+StartLimitBurst=3
 
 [Service]
-# www-data group and user need to be created before using these lines
-User=www-data
-Group=www-data
-
-ExecStart=/usr/bin/oauth2-proxy --config=/etc/oauth2-proxy.cfg
+User=oauth2-proxy
+Group=oauth2-proxy
+Restart=on-failure
+RestartSec=30
+WorkingDirectory=/etc/oauth2-proxy
+ExecStart=/usr/bin/oauth2-proxy --config=/etc/oauth2-proxy/oauth2-proxy.cfg
```

Comment:
@adrienverge If I understand well, it's here that you inform the binary that it should get its config from a non-default location. And it matches the [location of the config file](https://github.com/adrienverge/copr-oauth2_proxy/pull/4/files#diff-75102123e6b64dbba483204a20d7798051557d66a4c02aa66243a737d7bc509bL49-R49) that you package in the RPM on the COPR repo? (I was a bit puzzled at first because `oauth2-proxy's` docs [still mention the same `/etc/oauth2-proxy.cfg`](https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview/#:~:text=An%20example%20oauth2%2Dproxy.cfg%20config%20file%20is%20in%20the%20contrib%20directory.%20It%20can%20be%20used%20by%20specifying%20%2D%2Dconfig%3D/etc/oauth2%2Dproxy.cfg), so I didn't see immediately why you changed the paths in the Ansible files)

Reply:
Exactly, this is it.

Actually it's not directly me who changed the config path from `/etc/oauth2-proxy.cfg` to `/etc/oauth2-proxy/oauth2-proxy.cfg`: I just updated this systemd file from [the latest example upstream](https://github.com/oauth2-proxy/oauth2-proxy/blob/e00c7a7/contrib/oauth2-proxy.service.example#L14), which was updated last October (commit https://github.com/oauth2-proxy/oauth2-proxy/commit/e00c7a7).

I don't won't why they did that, but I "just took the file" and adapted the rest. Meanwhile, I didn't realize that their documentation still mentions the "old" `/etc/oauth2-proxy.cfg`. Anyway, I think this was a good move, because we can put other oauth2-proxy config files in the common dir `/etc/oauth2-proxy/` (for example `authenticated_emails_file = "/etc/oauth2-proxy/authorized_emails"`).

Thanks for pointing this out :slightly_smiling_face: